pulp_ansible/app/tasks/collections.py

```diff
@@ -296,13 +297,17 @@ async def run(self):
 
                 artifact = metadata["artifact"]
 
+                extradata = dict(docs_blob_url=metadata["docs_blob_url"])
+
                 d_artifact = DeclarativeArtifact(
                     artifact=Artifact(sha256=artifact["sha256"], size=artifact["size"]),
                     url=url,
                     relative_path=collection_version.relative_path,
                     remote=self.remote,
                     deferred_download=self.deferred_download,
+                    extra_data=extradata,
```

Comment:
Since the docs blob data follows the `CollectionVersion` I believe this extra_data should live on `DeclarativeVersion` with [this](https://github.com/pulp/pulpcore/blob/b94abd64d76ea4554e6750ff38ce458eaa888cc8/pulpcore/plugin/stages/models.py#L117) instead.

Reply:
This would basically move this to L#310